src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,100 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.UserAgeException;
+import core.basesyntax.exception.UserLoginExistsException;
+import core.basesyntax.exception.UserLoginLengthException;
+import core.basesyntax.exception.UserPasswordLengthException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Test
+    @DisplayName("User register with null login")
+    void register_nullLogin_throwNullPointerException() {
+        User userWithNullLogin = new User(null, "1234567", 45);
+        assertThrows(NullPointerException.class,
+                     () -> registrationService.register(userWithNullLogin));
```

Comment:
According to the checklist, you should not use `NullPointerException`. Instead, your code should throw a specific exception related to the validation, such as `IllegalArgumentException` or a custom exception, to clearly indicate the issue is with the user input validation.